env.js

```diff
@@ -45,6 +44,66 @@ function config ( ) {
   env.devicestatus_collection = readENV('MONGO_DEVICESTATUS_COLLECTION', 'devicestatus');
 
   env.enable = readENV('ENABLE');
+
+  env.DISPLAY_UNITS = readENV('DISPLAY_UNITS', 'mg/dl');
+
+  // parse defaults here, expect format `key1=value1 key2=value2` split on space, ignore commas
+  var rawdefaults = readENV('DEFAULTS','');
+  env.defaults = { // currently supported keys must defined be here
```

Comment:
The original idea for the defaults env was to make it a take arbitrary key=value pairs.  We've since dissed on gitter and have decided that we don't want to to that.